core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -497,6 +497,10 @@ class GroupCoordinator(val brokerId: Int,
     }
   }
 
+  def handleDeletedPartitions(tpList: Seq[TopicPartition]) {
+    for (tp <- tpList) groupManager.cleanupGroupMetadata(Some(tp))
```

Comment:
One query:  For commit pending offsets, we are appending to log, but we are not adding to  cache. Will this cause any issue during failover/or any other scenario?